UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs

```diff
@@ -108,6 +109,15 @@ public class CommonTraits : SingletonScriptableObject<CommonTraits>
 	/// <summary>
 	/// Do not use this list to get references to traits, locally reference them in your scripts instead!
 	/// </summary>
-	[Obsolete]
 	public List<ItemTrait> everyTraitOutThere = new List<ItemTrait>();
+
+	public ItemTrait GetFromIndex(short index)
+	{
+		return everyTraitOutThere.ElementAt(index);
```

Comment:
if possible, we need to find a different solution for this because using that field should be a sin.